vic/drivers/image/src/vic_force.c

```diff
@@ -89,7 +104,7 @@ vic_force(void)
     for (j = 0; j < NF; j++) {
         d3start[0] = global_param.forceskip[0] + global_param.forceoffset[0] +
                      j;
-        get_scatter_nc_field_double(filenames.forcing[0],
+        get_scatter_nc_field_double(&(filenames.forcing[0]),
```

Comment:
Why are you doing `&(filenames.forcing[0])`? Isn't `filenames.forcing[0]` already a string (`char *`)? I think that you can revert all these to `filenames.forcing[0]`.